apps/mull-ui/src/app/components/event-card/event-card.scss

```diff
@@ -3,6 +3,7 @@
 .event-card-container {
   position: relative;
   box-shadow: 0 0.25rem 0.25rem rgba(0, 0, 0, 0.25);
+  margin-bottom: 2rem;
```

Comment:
I don't think this margin should always be there. Whatever page is using the event card can decide how spaced out it is from other components.

It doesn't make much of a difference, but I think the component should decide how it looks and is formatted, but shouldn't worry about its positioning or spacing with other components, except in special cases like the navigation components.

Reply:
Good point. Didn't notice it's used in other places. with different spacing.

I moved this into the `discover.scss`. The inline style was introducing a weird workflow where the component was handling some styling and the scss was handling other parts of the styling and was forced to use `!important` to overwrite the inline style.

We can still use the `style` prop on event card for cases like if we want to be display none when some variable is false.
Let me know what you think.

Reply:
yeah sorry about that, the style thing was probably a remnant of me setting it up as a placeholder demo